CP01CNSM/CP01CNSM_R00008_ingest.csv

```diff
@@ -1,47 +1,47 @@
+mi.dataset.driver.presf_abc.presf_abc_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/instruments/dcl36/PRESF*/*.hex,CP01CNSM-MFD35-02-PRESFB000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl36/velpt2/*velpt*.log,CP01CNSM-MFD35-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/instruments/dcl36/VELPT*/*.aqd,CP01CNSM-MFD35-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl36/pco2w/*.pco2w.log,CP01CNSM-MFD35-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/instruments/dcl36/PCO2W*/SAMI_C0098_290318.txt,CP01CNSM-MFD35-05-PCO2WB000,recovered_inst,Available,not returning any data files when parsed on my machine
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl36/phsen2/*.phsen2.log,CP01CNSM-MFD35-06-PHSEND000,recovered_host,Available,returning an error when parsed on my machine
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/instruments/dcl36/PHSEN*/SAMI_P0097_290318.txt,CP01CNSM-MFD35-06-PHSEND000,recovered_inst,Available,not returning any data files when parsed on my machine
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl37/syslog/*.syslog.log,CP01CNSM-MFD37-00-DCLENG000,recovered_host,Available,
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl37/optaa2/*.optaa2.log,CP01CNSM-MFD37-01-OPTAAD000,recovered_host,Not Deployed,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl37/ctdbp2/*ctdbp2.log,CP01CNSM-MFD37-03-CTDBPD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/instruments/dcl37/CTDBP*/SBE*.hex,CP01CNSM-MFD37-03-CTDBPD000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl37/dosta2/*dosta2.log,CP01CNSM-MFD37-04-DOSTAD000,recovered_host,Available,not returning any data files when parsed on my machine
+mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/cg_data/dcl37/zplsc/*.zplsc.log,CP01CNSM-MFD37-07-ZPLSCC000,recovered_host,Available,
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00008/instruments/dcl37/ZPLSC*/DATA/*/*.01A,CP01CNSM-MFD37-07-ZPLSCC000,recovered_inst,Available,test parser

```

Comment:
Commented out this line because we cannot correctly ingest ZPLSC recovered instrument data yet

Reply:
This parser is OBE, this line should be deleted.